RecoLocalCalo/HGCalRecAlgos/src/HGCalImagingAlgo.cc

```diff
@@ -505,5 +544,58 @@ void HGCalImagingAlgo::shareEnergy(const std::vector<KDNode>& incluster,
     //update convergance parameter outside loop
     diff = std::sqrt(diff2);
     //std::cout << " iteration = " << iter << " diff = " << diff << std::endl;
-  } 
+  }
+}
+
+void HGCalImagingAlgo::computeThreshold() {
+
+  if(initialized) return;
+  const std::vector<DetId>& listee(rhtools_.getGeometry()->getValidDetIds(DetId::Forward,ForwardSubdetector::HGCEE));
+  const std::vector<DetId>& listfh(rhtools_.getGeometry()->getValidDetIds(DetId::Forward,ForwardSubdetector::HGCHEF));
+  //  const std::vector<DetId>& listbh(rhtools_.getGeometry()->getValidDetIds(DetId::Hcal,HcalSubdetector::HcalEndcap));
+  
+  std::vector<double> dummy;
+  dummy.resize(700,0);
```

Comment:
This number is related to the maximum number of wafers, I contacted geometry experts to figure out if there's a better way to determine it.